src/app/static/src/store/state.ts

```diff
@@ -4,4 +4,5 @@ export interface RootState {
   count: number
   apiInfo: ApiInfo | null
   metadata: Metadata | null
+  results: unknown | null
```

Comment:
Out of curiosity, is this different to the following?
```
  results?: unknown
```

Reply:
I think `results?` indicates that the field can be missing i.e. undefined, so it would actually be equivalent to `results: unknown | undefined`